code/game/objects/structures/statues.dm

```diff
@@ -203,6 +203,12 @@
 		PlasmaBurn(2500)
 	else if(istype(Proj,/obj/item/projectile/ion))
 		PlasmaBurn(500)
+	if(Proj.firer)
+		message_admins("Plasma statue ignited by [key_name_admin(Proj.firer)](<A HREF='?_src_=holder;adminmoreinfo=\ref[Proj.firer]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[Proj.firer]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
+		log_game("Plasma statue ignited by [key_name(Proj.firer)] in ([x],[y],[z])")
+	else
+		message_admins("Plasma statue ignited by [Proj]. No known firer.(<A HREF='?_src_=holder;adminmoreinfo=\ref[Proj.firer]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[Proj.firer]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
+		log_game("Plasma statue ignited by [Proj] in ([x],[y],[z]). No known firer.")	
```

Comment:
this is going to message the admins even if the bullet_act() doesn't call PlasmaBurn() i.e. even when it doesn't ignite the statue.